packages/x-components/src/design-system/components/suggestion-group/default.tokens.scss

```diff
@@ -11,6 +11,19 @@
   --x-size-padding-left-suggestion-group-default: var(--x-size-padding-left-suggestion-default);
   --x-size-gap-suggestion-group-default: var(--x-size-gap-suggestion-default);
 
+  // border
```

Comment:
you've added all these tokens, but they are not used anywhere
you have to use them in the `suggestion-group/default.scss`

Reply:
and if you added the border width, you have to add a token for the color too